web/cars/next.config.js.encrypted

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Should we update this file with the new env var name ?

Reply:
done